opm/parser/share/keywords/000_Eclipse100/G/GCONPROD

```diff
@@ -1,10 +1,10 @@
 {"name" : "GCONPROD" , "sections" : ["SCHEDULE"], "items" : [
         {"name" : "GROUP" , "value_type" : "STRING" },
         {"name" : "CONTROL_MODE" , "value_type" : "STRING" , "default" : "NONE"},
-        {"name" : "OIL_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
-        {"name" : "WATER_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
-        {"name" : "GAS_TARGET" , "value_type" : "DOUBLE" , "dimension" : "GasSurfaceVolume/Time", "default" : 0},
-        {"name" : "LIQUID_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : 0},
+        {"name" : "OIL_TARGET" , "value_type" : "DOUBLE" , "dimension" : "LiquidSurfaceVolume/Time", "default" : -999e100},
```

Comment:
I have strived hard to keep such *raving-mad-values-as-indicators-of-default* out of the Json configuration; the goal is that any default value mentioned in the Json configuration should be usable as is. If the client is interested in knowing whether the default has been applied it should use the `item->defaultApplied()` method instead of comparing with a more or less funny value.

Even in in this particular case I do not think the value `-999e100` is required in any way, the code in `Group.cpp` initializes the rate vectors with the `INVALID_GROUP_RATE` value - and as long as there is no `GCONPROD` keyword in the deck the value given here will not even be consulted.

I will back off if your empirical evidence shows otherwise, but if that is not the case I would like these defaults to remain at 0.